raiden/tests/scenarios/bf5_join_and_leave.yaml

```diff
@@ -0,0 +1,165 @@
+            # node3 should automatically deposit funds / 5 in the channels it opens
+            - assert: {from: 3, to: 0, total_deposit: 200_000_000_000_000_000, balance: 200_000_000_000_000_000, state: "opened"}
+            - assert: {from: 3, to: 1, total_deposit: 200_000_000_000_000_000, balance: 200_000_000_000_000_000, state: "opened"}
+            - assert: {from: 3, to: 2, total_deposit: 200_000_000_000_000_000, balance: 200_000_000_000_000_000, state: "opened"}
+      - serial:
+          name: "Make transfers to make sure that path finding works with join"
+          tasks:
+            - transfer: {from: 3, to: 0, amount: 1_000_000_000_000_000, lock_timeout: 30}
+            - transfer: {from: 3, to: 1, amount: 1_000_000_000_000_000, lock_timeout: 30}
+            - transfer: {from: 3, to: 2, amount: 1_000_000_000_000_000, lock_timeout: 30}
+            - transfer: {from: 2, to: 0, amount: 1_000_000_000_000_000, lock_timeout: 30}
+            - transfer: {from: 2, to: 1, amount: 1_000_000_000_000_000, lock_timeout: 30}
+            - transfer: {from: 2, to: 3, amount: 1_000_000_000_000_000, lock_timeout: 30}
+            - transfer: {from: 0, to: 2, amount: 1_000_000_000_000_000, lock_timeout: 30}
+            - transfer: {from: 0, to: 3, amount: 1_000_000_000_000_000, lock_timeout: 30}
+            - transfer: {from: 1, to: 2, amount: 1_000_000_000_000_000, lock_timeout: 30}
+            - transfer: {from: 1, to: 3, amount: 1_000_000_000_000_000, lock_timeout: 30}
+      - parallel:
+          name: "Assert after transfers. Should be status quo"
+          tasks:
+            - wait_blocks: 2
+            - assert: {from: 0, to: 1, total_deposit: 1_000_000_000_000_000_000, balance: 1_000_000_000_000_000_000, state: "opened"}
+            - assert: {from: 0, to: 2, total_deposit: 0, balance: 0, state: "opened"}
+            - assert: {from: 0, to: 3, total_deposit: 0, balance: 0, state: "opened"}
+            - assert: {from: 1, to: 0, total_deposit: 1_000_000_000_000_000_000, balance: 1_000_000_000_000_000_000, state: "opened"}
+            - assert: {from: 1, to: 2, total_deposit: 0, balance: 0, state: "opened"}
+            - assert: {from: 1, to: 3, total_deposit: 0, balance: 0, state: "opened"}
+            - assert: {from: 2, to: 0, total_deposit: 200_000_000_000_000_000, balance: 200_000_000_000_000_000, state: "opened"}
+            - assert: {from: 2, to: 1, total_deposit: 200_000_000_000_000_000, balance: 200_000_000_000_000_000, state: "opened"}
+            - assert: {from: 2, to: 3, total_deposit: 200_000_000_000_000_000, balance: 200_000_000_000_000_000, state: "opened"}
+            - assert: {from: 3, to: 0, total_deposit: 200_000_000_000_000_000, balance: 200_000_000_000_000_000, state: "opened"}
+            - assert: {from: 3, to: 1, total_deposit: 200_000_000_000_000_000, balance: 200_000_000_000_000_000, state: "opened"}
```

Comment:
So this is the assert that fails? If it fails shouldn't you find out why and fix it before merging?

Reply:
Indeed. I didn't expect anyone to look at this PR until tomorrow and also didn't add the "please review" tag to it. I made the PR so that I can get a second pair of eyes on the scenario and help me find out what's wrong.

Reply:
Then you should make a "Draft PR". By opening a normal PR a reviewer is automatically assigned to it and if tests pass reviews it.

Reply:
Thanks, will do in the future.

I cannot find any problems with the scenario though, so maybe it's not scenario related but a problem with Raiden.